sys-fs/zfs-kmod/zfs-kmod-2.2.7.ebuild

```diff
@@ -0,0 +1,207 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MODULES_INITRAMFS_IUSE=+initramfs
+inherit autotools flag-o-matic linux-mod-r1 multiprocessing
+
+DESCRIPTION="Linux ZFS kernel module for sys-fs/zfs"
+HOMEPAGE="https://github.com/openzfs/zfs"
+
+MODULES_KERNEL_MAX=6.10
```

Comment:
Isn't it 6.12?

Reply:
Yes. Let me fix that...

Reply:
Jeez, sorry. My git skills are not very good, and my attempt to amend the commit left some weird artifact in the file.